doc/user_guide/model.rst

```diff
@@ -420,12 +420,22 @@ Getting and setting parameter values and attributes
 :py:meth:`~.model.BaseModel.print_current_values` prints the value of the
 parameters of the components in the current coordinates.
 
+The current coordinates can be either set by navigating the :py:meth:`~.model.BaseModel.plot`, or specified by
+pixel indices in ``m.axes_manager.indices`` or as calibrated coordinates in
+``m.axes_manager.coordinates``.
+
 :py:attr:`~.component.Component.parameters` contains a list of the parameters
 of a component and :py:attr:`~.component.Component.free_parameters` lists only
 the free parameters.
 
-The value of a particular parameter can be accessed in the
-:py:attr:`~.component.Parameter.value`.
+The value of a particular parameter in the current coordinates can be 
```

Comment:
Wouldn't it be better to document ``Parameter.as_signal`` in the UG instead and move this to the source as comments for the developers? ``as_signal`` returns the same arrays but in a more user-friendly way. Users needing direct access to ``Parameter.map`` should be able to find their way by inspecting the code.

``map`` is not private because when I wrote this part of the code it hadn't yet crossed my mind that anybody else would use it, so I didn't follow any convention.

Reply:
I think it is still worth leaving this in the user guide, because I have seen users using map without going through the code...